### DISCUSSION TITLE
About calibration and Color Profiles (Win)

### DISCUSSION THREAD

**Discussion by ghost** [Beep6581/RawTherapee#6707] [Q&A] [answered]
Hey, this is the first time i have my Spyder X to calibrate my monitor and i really dislike the calibrated colors :) It is more that warm i like for all the other business tasks i have to do.

I have 2 concrete profiles:

- my camera shots in AdobeRGB
- my monitor is calibrated

After using the Spyder Tool, it automatically saves my profile to Windows and sets it. Now my colors are warm.

In RT we can set:

![image](https://user-images.githubusercontent.com/126161988/224570834-e93163af-8866-4947-a241-f19e88881d46.png)

So, **must** i set the color profile as active **in Windows and in RT** or is _one of them_ enough?

<ins>I just want to set the color profile in RT</ins> and not for all in Windows.

So, if i drop the profile from Windows and **set my Monitor ICC only in RT**, is that ok?

If i drop the profile from Windows, changes in RT are minimal. If i set the profile in Windows, the differences are much much bigger.

What exactly does the profile in RT that it must be set anyway / besides Windows icm?

**Comment by Lawrence37** [comment, marked as answer]
The correction profile should be applied exactly once, so either by Windows OR by RawTherapee, never both.

For questions that are not related to the development of RawTherapee, I recommend starting topics in the Pixls forum to reach a larger audience of knowledgeable users.
